src/main/scala/latis/server/CapabilitiesService.scala

```diff
@@ -10,6 +13,30 @@ class CapabilitiesService[F[_]: Effect] extends Http4sDsl[F] {
   val service: HttpService[F] =
     HttpService[F] {
       case GET -> Root / "capabilities" =>
-        Ok("Hello from HAPI!")
+        val capabilities =
+          Capabilities("2.0", Status.`1200`, List("binary", "csv", "json"))
+        Ok(capabilities.asJson)
+    }
+}
+
+/**
+ * Response from the `/capabilities` endpoint.
+ *
+ * @param version version of HAPI
+ * @param status HAPI status object
+ * @param formats list of supported formats
+ */
+final case class Capabilities(
+  version: String,
+  status: Status,
+  formats: List[String]
```

Comment:
Note for the future:

I can imagine defining something like

```scala
sealed abstract trait OutputFormat
final case object CSV extends OutputFormat
final case object JSON extends OutputFormat
final case object BIN extends OutputFormat
final case class Custom(name: String) extends OutputFormat
```

This is probably better than accepting arbitrary strings, and we could enforce the requirement that custom output formats start with `x_` in the response.